root/release/merge.tt

```diff
@@ -19,30 +19,32 @@
                     <th>[% l('Label') %]</th>
                     <th>[% l('Catalog#') %]</th>
                     <th>[% l('Barcode') %]</th>
+                    [%- remove_from_merge_header(to_merge) -%]
                 </tr>
             </thead>
             <tbody>
-            [%- FOR release=to_merge -%]
+            [%- FOR entity=to_merge -%]
```

Comment:
Renaming release to entity for consistency (and for ease of reusing the current macro).